src/plugins/syscalls/syscalls.cpp

```diff
@@ -130,9 +130,36 @@ static event_response_t linux_cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
     return 0;
 }
 
+static event_response_t win_ret_cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
```

Comment:
I think it would make sense to only print output in this callback so that everything would get printed in one block.